site/_includes/themes/apache-clean/_navigation.html

```diff
@@ -5,8 +5,8 @@
     <!--h1>{{ site.data.project.name }}</h1-->
     <p>{{ site.data.project.description }}</p>
 
-    {% if site.data.project.latest_release %}
-    <a href="{{ site.data.project.download }}" class="btn btn-primary btn-lg bigFingerButton" role="button">Download {{ site.data.project.name }} {{site.data.project.latest_release}}</a></p>
+    {% if site.data.project.download %}
+      <a href="{{ site.data.project.download }}" class="btn btn-primary btn-lg bigFingerButton" role="button">Download {{ site.data.project.name }} Latest Releases</a></p>
```

Comment:
I think we should add two buttons, one for Bahir Spark and one for Bahir Flink (once available) and keep the version in the button text(s)